TabulateSmarterTestAdminPackage/Processors/Specification/TestSpecification/Administration/ItemPool/TestItem/PassageRefProcessor.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Xml.XPath;
+using TabulateSmarterTestAdminPackage.Common.Enums;
+using TabulateSmarterTestAdminPackage.Common.Validators;
+using TabulateSmarterTestAdminPackage.Utility;
+
+namespace TabulateSmarterTestAdminPackage.Processors.Specification.TestSpecification.Administration.ItemPool.TestItem
+{
+    internal class PassageRefProcessor : Processor
+    {
+        internal PassageRefProcessor(XPathNavigator navigator) : base(navigator) {}
+
+        private string PassageRef { get; set; }
+
+        public override bool Process()
+        {
+            return IsValidPassageRef();
+        }
+
+        // This is not required, but if the element is there, it should have a value
+        internal bool IsValidPassageRef()
+        {
+            var validators = new ValidatorCollection
+            {
+                new RequiredStringValidator(ErrorSeverity.Benign),
+                new MaxLengthValidator(ErrorSeverity.Benign, 100)
+            };
+            PassageRef = Navigator.Value;
+            if (validators.IsValid(PassageRef))
```

Comment:
Should this be using the IsValid() method that has the 2nd parameter for isRequired?